04-single-hop-6lowpan-icmp/test_spec04.py

```diff
@@ -40,8 +48,7 @@ def test_task01(riot_ctrl):
                 count=1000, interval=20, packet_size=0)
     assert res['stats']['packet_loss'] < 10
 
-    assert pktbuf(pinged).is_empty()
-    assert pktbuf(pinger).is_empty()
+    check_pktbuf((pinged, pinger))
```

Comment:
and

```suggestion
    check_pktbuf(pinged, pinger)
```

in those calls.